src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/ActiveDirectoryNativeAuthenticationProvider.cs

```diff
@@ -52,6 +52,7 @@ internal class ActiveDirectoryNativeAuthenticationProvider : SqlAuthenticationPr
             {
                 result = await app.AcquireTokenInteractive(scopes)
                   .WithUseEmbeddedWebView(true)
+                  .WithLoginHint(parameters.UserId)
```

Comment:
Do we need this change in netcore as well?

Reply:
We don't support Interactive in .NET Core yet.